src/pages/signin/LoginForm/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import BaseLoginForm from './BaseLoginForm';
+
+const propTypes = {
+    /** Function used to scroll to the top of the page */
+    scrollPageToTop: PropTypes.func,
+};
+const defaultProps = {
+    scrollPageToTop: () => {},
```

Comment:
Let's update this to `undefined` to get aligned with the others.

Reply:
👍 updated @mollfpr